src/_sass/lh.scss

```diff
@@ -1,47 +1,174 @@
-$pink-lh: #F70D88;
-$yellow-lh: #F9DF25;
-$blue-lh: #01ADEF;
+$pink-lh: #f70d88;
+$yellow-lh: #f9df25;
+$blue-lh: #01adef;
 $gray-lh: #222;
 $light-gray-lh: #555;
-$near-white-lh: #EEE;
+$near-white-lh: #eee;
+
+:root {
+  --replace-offset: 50%;
+  --replace-offset-2: calc((100% - var(--replace-offset)) * -1);
+  --header-background: #f70d88;
+  --header-background-invert: #f70d88;
+  --header-text-color: white;
+  --header-text-color-invert: white;
+}
```

Comment:
Why :root tho? We're on SCSS, we can define SCSS native variables, just like above. We could keep it for the `--replace-offset` variables, the other ones look like they could be moved

Reply:
I modify these variables with javascript and I don't know if SCSS variables could handle this operation. For instance, I'll let these variables in `:root`